keel-core/src/main/kotlin/com/netflix/spinnaker/keel/model/Moniker.kt

```diff
@@ -18,14 +18,14 @@ package com.netflix.spinnaker.keel.model
 import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import com.netflix.frigga.Names
 
 @JsonInclude(NON_NULL)
 data class Moniker(
   val app: String,
   val stack: String? = null,
   val detail: String? = null,
-  val sequence: String? = null,
-  val cluster: String? = null
+  val sequence: String? = null
```

Comment:
The `cluster` field as I understand it is always derived from the others, so it doesn't seem to make sense for us to store it. We never use it anyway.